.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
Thanks Layne, I'll fix my mistake this morning.

On Thu, 28 Jul 2016, 09:20 laynecsmall, <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/operationzombie/Embedded-devices/pull/2#discussion_r72526091>
> :
>
> > @@ -8,6 +8,9 @@ install:
> >  notifications:
> >    slack:
> >      rooms:
> > -      - undeadbaby:$SLACK_KEY#random
> > +    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
>
> You're an owner on the github operationzombie organization, so you've got
> control over all the repos in there.
>
> I've manually added you as admin for those two repos, so hopefully it
> should be sorted.
> (You might need to get travis to re-sync your repos to recognize new
> permissions though)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/operationzombie/Embedded-devices/pull/2/files/e8360690be58cb5f7b920e5dcbda16e808fad5b8#r72526091>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARK0lyyGm2ZBTdS3z6LBPF-7lcToxKOkks5qZ8uXgaJpZM4JVr6v>
> .
>